ibis/expr/operations/arrays.py

```diff
@@ -266,3 +266,60 @@ class TimestampRange(Range):
     start: Value[dt.Timestamp]
     stop: Value[dt.Timestamp]
     step: Value[dt.Interval]
+
+
+class ArrayAgg(Value):
```

Comment:
Should we rename to _ArrayAgg? I dont think this should be exposed outside this file

Reply:
I think this is fine to leave as is. `ibis.expr.operations` isn't public.